src/write_http.c

```diff
@@ -457,6 +523,13 @@ static int wh_write(metric_family_t const *fam, user_data_t *user_data) {
   case WH_FORMAT_INFLUXDB:
     status = wh_write_influxdb(fam, cb);
     break;
+  case WH_FORMAT_OTLP_PROTO:
```

Comment:
Nothing is setting/using `WH_FORMAT_OTLP_PROTO`, so IMHO it would be better to drop references to that completely from this PR.

Reply:
That's fair. Done.